var/spack/repos/builtin/packages/py-attmap/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyAttmap(PythonPackage):
+    """Key-value mapping that supports nested attribute-style access."""
+
+    homepage = "https://github.com/pepkit/attmap/"
+    pypi = "attmap/attmap-0.13.2.tar.gz"
+
+    version("0.13.2", sha256="fdffa45f8671c13428eb8c3a1702bfdd1123badb99f7af14d72ad53cc7e770de")
+
+    depends_on("python@3.5:", type=("build", "run"))
```

Comment:
Where do you see this? Honestly I would just remove it, Spack no longer supports Python 3.4 and older anyway

Reply:
done!